tests/test_protobuf.py

```diff
@@ -0,0 +1,53 @@
+import json
+
+from utils import weblog, interfaces, context, missing_feature, features
+
+
+# this test relies on the proto file at utils/build/docker/common/message.proto
+@features.datastreams_monitoring_protobuf_schema_tracking
+class Test_Protobuf:
+    def setup_protobuf(self):
+        self.serialization_response = weblog.get("/protobuf/serialize")
+        self.deserialization_response = weblog.get(f"/protobuf/deserialize?msg={self.serialization_response.text}")
+
+    @missing_feature(
+        context.library in ["ruby", "cpp", "golang", "nodejs", "php", "python"],
+        reason="no schema tracking for protobuf yet",
+    )
+    @missing_feature(context.library < "dotnet@3.12.0", reason="wrong schema.name written")
+    @missing_feature(context.library < "java@1.47.0", reason="bug on extraction of nested schemas")
+    def test_protobuf(self):
```

Comment:
we should define all these missing_feature in the manifests

Reply:
oh you mean the library versions ? Or also the languages where it's not available yet ?

Reply:
both cases. I think we can safely use this python class without such annotations.